### ISSUE TITLE
Source/PR Request (I guess)

### ISSUE THREAD

**Issue by EoF-1141** [MrFawkes1337/TheBetterRoles#1]
If you plan on fixing all the bugs, why don't you do what everyone does and submit a PR.

"Lag Issue" was issued, why don't you submit a PR for it or respond to the Issue, when you have it fixed.

About the settings - not everyone likes to play the same way. People like different options and some things are just playercount dependent (Jackal/Sidekick). Removing all options isn't a good "solution" imo.

**Comment by MrFawkes1337** [comment]
*sigh*, I guess i'll address this now, instead of Messaging Eisbison about it like I planned. 

To answer your first question about why I didn't submit a pull request:

    During testing of all versions of the Other Roles from 2020.12.9s to latest, different roles added different bugs that have gone 
    unfixed. For example, the lag issue. While I fixed the issue, it's messy code, so I plan to clean it all up before I even think about   
    a pull request. I also have a few bugs I wanna squash in previous updates before I release, which you don't support. 
    
As for the settings, This is more for me and friends, and if others use it, and request the changes back it's a simple thing to add. We just have a preference for certain things, which is why stuff is preconfigured.

I also changed several things, like the Engineer and Repairs, the Assassin and the guesser being two separate roles, instead of one for preference, the Phantom and Miner which were intended Pull requests for ToR as well, once I had everything tested. New custom button art a friend designed. 

TL;DR, Source will be posted along with release, as well as a pull request for the above fixes / features. There was never an intention to leave anyone out, nor any ill intentions meant by the fork, just sheer personal preference, which was stated in the ReadMe where I state (Sorry Eisbison, It's just how I feel.)

Closing for now, feel free to open another when the project is released if you have issues. Should be within a week.
